payjoin/Cargo.toml

```diff
@@ -26,6 +26,7 @@ v2 = ["_core", "bitcoin/serde", "hpke", "dep:http", "bhttp", "ohttp", "serde", "
 #[doc = "Functions to fetch OHTTP keys via CONNECT proxy using reqwest. Enables `v2` since only `v2` uses OHTTP."]
 io = ["v2", "reqwest/rustls-tls"]
 _danger-local-https = ["reqwest/rustls-tls", "rustls"]
+multi-party = ["v2", "psbt-merge"]
```

Comment:
This feature should replace `psbt-merge`, not add a new one, since `psbt-merge` is useless otherwise.

As discussed in https://github.com/payjoin/rust-payjoin/pull/523#discussion_r1940434050

I also want it to be named `multiparty` (and have the mod named `multiparty`) for brevity and consistency.

Reply:
done